airgun/views/host_new.py

```diff
@@ -344,6 +344,34 @@ class networking_interfaces(Card):
         class networking_interface(Card):
             pass
 
+        @View.nested
+        class virtualization(Card):
+            ROOT = './/article[contains(@data-ouia-component-id, "card-template-Virtualization")]'
+
+            datacenter = Text('.//div[contains(@class, "pf-c-description-list__group")][1]//dd')
+            cluster = Text('.//div[contains(@class, "pf-c-description-list__group")][2]//dd')
```

Comment:
I'm not a fan of using hard number locators, these are so fragile that in case if any new field/label is added, this will disrupt all other locators. I see this pattern is been used across the whole class which I really think needs to improve in Airgun.

e.g . `$x("//div[@class='pf-c-description-list__group']//span[text()='Host group']/parent::*/following-sibling::*")`

you also could have looked at the existing custom card, https://github.com/SatelliteQE/airgun/blob/88a9f225088ca84ec865e10926522eb2fbf58ea8/airgun/views/host_new.py#L74 nicely return all the attributes and not need to add all the locators.

Reply:
Good suggestion, I prefer this too. I didn't see that existing utility. I'll make an issue, and get a fix raised for this in master, and then merged into 6.13.z - lets merge this, and then I'll fix it shortly.